paasta_tools/kubernetes_tools.py

```diff
@@ -179,6 +179,7 @@
 
 DEFAULT_USE_PROMETHEUS_CPU = False
 DEFAULT_USE_PROMETHEUS_UWSGI = True
+DEFAULT_USE_PROMETHEUS_PISCINA = True
```

Comment:
(there's a comment below talking about use_prometheus - remove this constant once the usage below is cleaned up)